OWML.Common/Menus/IModPromptButton.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace OWML.Common.Menus
 {
-    public interface IModPromptButton : IModButton
+    public interface IModPromptButton : IModTitleButton
     {
         string DefaultTitle { get; }
```

Comment:
Still don't wanna use Title from TitleButton?

Reply:
Wasn't really neccessary to override... maybe only to make it keep LocalizedText if it matches DefaultTitle